test/optimus/pr/ValidateOptimus.wdl

```diff
@@ -54,11 +58,18 @@ workflow ValidateOptimus {
              metric_and_index_validation_result = ValidateMetrics.result,
              loom_validation_result = ValidateLoom.result
     }
+
+    output {
+
```

Comment:
This is OK for now, but an improvement would be returning the validation results as outputs.

Reply:
I have to write an empty output section, as otherwise womtools validation would fail at this point. So just leave it for later addition.

Reply:
I saw that the master version doesn't return anything, so I didn't add those validation results returned from subtasks to this output section. I thought it was intentional by Optimus team.